server/game/cards/02_SHD/units/EnfysNestMarauder.ts

```diff
@@ -0,0 +1,23 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer } from '../../../core/Constants';
+
+export default class EnfysNestMarauder extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '8107876051',
+            internalName: 'enfys-nest#marauder'
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.addConstantAbility({
+            title: 'When a friendly unit is attacking using Ambush, the defender gets -3/-0',
+            targetController: RelativePlayer.Opponent,
+            matchTarget: (card, context) => card.isUnit() && card.isInPlay() && card.isDefending() && card.activeAttack.attacker.controller === context.source.controller && card.activeAttack.isAmbush,
```

Comment:
nit: since there's a ton of conditions, do you mind breaking them out on separate lines for readability. Great job though, this is the exact right set of conditions